### ISSUE TITLE
Accessibility: More than one (h1) tag on a page

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#9]
WCAG 2.0 - Success Criteria - 1.3.1 Level (A) violation
1. Go to http://data.ottawa.ca/group
2. Observe below there must be only one (h1) label per page.

Resolution: Remove the second (h1) label

![h two](https://cloud.githubusercontent.com/assets/5271253/8034701/2e35fdd8-0db8-11e5-8bf8-7f19bdfba46d.JPG)
![h1_home_fr](https://cloud.githubusercontent.com/assets/5271253/8058450/793a7b9a-0e87-11e5-9977-320ea7d7cb94.JPG)

**Comment by deniszgonjanin** [comment]
@Dawnzar ready for review

**Comment by Dawnzar** [comment]
Retested there should only be one h1 tag per page.
![opendata_mon](https://cloud.githubusercontent.com/assets/5271253/8159995/fa480fac-1339-11e5-86ef-7cbb73d687ee.JPG)
@deniszgonjanin
